src/appmixer/utils/csv/AddRows/AddRows.js

```diff
@@ -35,10 +26,18 @@ module.exports = {
                 );
             }
         }
-        const savedFile = await processor.addRows({ rows: rowsArray }, (idx, currentRow, isEndOfFile) => {
-            return isEndOfFile;
+
+        // check if it withHeaders or not(if rows is an array of array string or array of objects)
```

Comment:
Improve the comment. The label from removed input seems fine.
```suggestion
        // true if the first row represents column names (CSV header) and you want to use them to identify the columns(If true, make sure you pass array of row objects i.e [{\"column1\": \"value1\", \"column2\": \"value2\"}...]).
```